### DISCUSSION TITLE
What are the default user name and password?

### DISCUSSION THREAD

**Discussion by wonderfuldh** [mikopbx/Core#389] [General ( Обо всём )]
I have downloaded below
 [mikopbx-2022.2.102-x86_64.iso](https://github.com/mikopbx/Core/releases/download/2022.2.102/mikopbx-2022.2.102-x86_64.iso)

I have successfully set up the VM in Virtual Box and can see the web UI of MikoPBX. However, when I tried to login using admin / admin The system just say 'Authorization error'.

What is the reason? Is the default user name and password really 'admin / admin'?

**Comment by boffart** [comment]
the default password should work. try to reset the password through the PBX console
<img width="717" alt="mikopbx" src="https://user-images.githubusercontent.com/5922739/182134674-01ec7b6b-3bb0-451b-9946-fa874f30ecb1.png">
